content/en/monitors/types/process.md

```diff
@@ -19,21 +19,44 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended to:
```

Comment:
Feels like a bit of an unfinished leading sentence, maybe
```suggestion
Live Process Monitors are best used in the following scenarios:
```

Reply:
Actually `recommend to:` was @alai97's suggestion: https://github.com/DataDog/documentation/pull/21167#discussion_r1440918292.

But yeah, I agree with your suggestion. I will update it